### PR TITLE
Rem 543

### PR DESCRIPTION
**Pull request by Alinadimmokdad**
iv-rebuildInventory/
rebuild page , dont contain a get request

### CODE REVIEW

**Comment by imtithalhalalo** [comment]
Screenshot 2024-09-05 144053
 
the page is not added yet

**Comment by imtithalhalalo** [comment]
should the form clear after rebuild or no?

File: src/pages/iv-rebuildInventory/form/IvRebuild.js [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+import CustomDatePicker from 'src/components/Inputs/CustomDatePicker'
+import { ResourceLookup } from 'src/components/Shared/ResourceLookup'
+import { InventoryRepository } from 'src/repositories/InventoryRepository'
+
+export default function RebuildForm({ _labels, access }) {
+  const { postRequest } = useContext(RequestsContext)
+  const { platformLabels } = useContext(ControlContext)
+
+  const { formik } = useForm({
+    initialValues: { year: '', itemId: 0, itemName: '', date: new Date(), recordId: 'N/A' },
```
Comment (Ghinwa-Arab) [inline src/pages/iv-rebuildInventory/form/IvRebuild.js:23]: 1- It's allowed to set recordId to N/A hard coded?! keep it empty string or null
2- Remove itemName

Comment (Alinadimmokdad) [reply src/pages/iv-rebuildInventory/form/IvRebuild.js:23]: in case of rebuild we set the record N/A

**Review by Ghinwa-Arab — APPROVED** [review]

**Pull request merged**